src/plugins/interactive-media/interactive-media-editor.js

```diff
@@ -74,13 +74,15 @@ function InteractiveMediaEditor({ content, onContentChanged }) {
 
   const changeContent = newContentValues => {
     const newContent = { ...content, ...newContentValues };
-    const newDisableVideo = shouldDisableVideo(newContent.sourceUrl);
+    const shouldDisableVideoOnNewUrl = shouldDisableVideo(newContent.sourceUrl);
+    const shouldDisableVideoOnOldUrl = shouldDisableVideo(content.sourceUrl);
 
-    const reEnableVideo = !content.showVideo;
-    newContent.showVideo = newDisableVideo ? false : reEnableVideo || newContent.showVideo;
-    newContent.posterImage = newDisableVideo ? { sourceUrl: '' } : newContent.posterImage;
+    const autoEnableVideo = !!shouldDisableVideoOnOldUrl && !shouldDisableVideoOnNewUrl;
+    const autoDisableVideo = !!shouldDisableVideoOnNewUrl;
+    newContent.showVideo = autoDisableVideo ? false : autoEnableVideo || newContent.showVideo;
+    newContent.posterImage = autoDisableVideo ? { sourceUrl: '' } : newContent.posterImage;
 
-    setDisableVideo(newDisableVideo);
+    setDisableVideo(autoDisableVideo);
     onContentChanged(newContent);
   };
```

Comment:
"Shouldn't we do this for all 3 plugins in this situation?" *asked Andreas* 
" 😱 oh, right!!" *said Adriana*

Reply:
Fixed!

Reply:
:rofl: